documentation/LLC-quickstart.md

```diff
@@ -0,0 +1,361 @@
+Getting Started - Generate the RLC rest-level client libraries.
+================================================================
+# Prerequisites
+You may refer to this [link](https://github.com/Azure/azure-sdk-for-js/blob/main/CONTRIBUTING.md#prerequisites) for the environment set up prerequisites in azure-sdk-for-js repository. 
+
+# How to generate RLC
```

Comment:
Maybe we can include some information on how to create the folder structure for the new project? Maybe point to copy the sdk/template and mention what needs to be updated like packageName, etc.